conda/environments/all_cuda-129_arch-aarch64.yaml

```diff
@@ -16,6 +16,7 @@ dependencies:
 - clang==20.1.4
 - cmake>=3.30.4
 - cramjam
+- croaring==4.3.5
```

Comment:
Looks like the required version is not available on conda so will likely have to remove it https://anaconda.org/conda-forge/croaring